setup.py

```diff
@@ -41,7 +41,7 @@
 
 _deps = [
     "numpy>=1.0.0,<=1.21.6",
-    "onnx>=1.5.0,<=1.12.0",
+    "onnx>=1.13.0,<=1.14.0",
```

Comment:
Is there a reason why the minimum of 1.5 needs to increase to 1.13? Would like to keep backwards compatibility for other OSs available

Reply:
good point, let me correct that!